platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/geometry/LatLngBounds.java

```diff
@@ -450,7 +450,9 @@ public LatLngBounds build() {
      */
     public Builder includes(List<LatLng> latLngs) {
       for (LatLng point : latLngs) {
-        latLngList.add(point);
+        if (!latLngList.contains(point)) {
```

Comment:
Could you extract this into a private method? The same exact piece of code is duplicated and could be simplified and used from both `includes` and `include` methods.